model/src/main/scala/uk/gov/ons/addressIndex/model/db/index/NationalAddressGazetteer.scala

```diff
@@ -0,0 +1,99 @@
+package uk.gov.ons.addressIndex.model.db.index
+
+import uk.gov.ons.addressIndex.model.server.response._
+
+case class NationalAddressGazetteer(
+  uprn: String,
+  postcodeLocator: String,
+  addressBasePostal: String,
+  latitude: String,
+  longitude: String,
+  easting: String,
+  northing: String,
+  organisation: String,
+  legalName: String,
+  classificationCode: String,
+  usrn: String,
+  lpiKey: String,
+  paoText: String,
+  paoStartNumber: String,
+  paoStartSuffix: String,
+  paoEndNumber: String,
+  paoEndSuffix: String,
+  saoText: String,
+  saoStartNumber: String,
+  saoStartSuffix: String,
+  saoEndNumber: String,
+  saoEndSuffix: String,
+  level: String,
+  officialFlag: String,
+  logicalStatus: String,
+  streetDescriptor: String,
+  townName: String,
+  locality: String
+) extends Formattable {
+
+  def formatAddress: String = {
```

Comment:
Checkout the formattable trait; moved the common repeated logic to one place.
Makes things easier to follow IMO, standard POJO-ee-ness.